src/hotspot/share/gc/shared/taskqueue.inline.hpp

```diff
@@ -49,7 +49,8 @@ inline GenericTaskQueueSet<T, F>::~GenericTaskQueueSet() {
 }
 
 template<class E, MEMFLAGS F, unsigned int N>
-inline void GenericTaskQueue<E, F, N>::initialize() {
+inline GenericTaskQueue<E, F, N>::GenericTaskQueue() : _last_stolen_queue_id(InvalidQueueId), _seed(17 /* random number */) {
+  assert(sizeof(Age) == sizeof(size_t), "Depends on this.");
```

Comment:
[pre-existing] This assert is over-constraining.  The real requirement is static-asserted in the Age class.

Reply:
Removed.